src/features/vote/PastRequests.tsx

```diff
@@ -60,15 +85,33 @@ const PastRequests: FC<Props> = ({ priceRounds, address, contract }) => {
               <th>Your Vote</th>
               <th>Earned Rewards</th>
               <th>Timestamp</th>
-              <th></th>
             </tr>
           </thead>
           <tbody>
             {pastRequests.map((el, index) => {
               return (
                 <tr key={index}>
                   <td>
-                    <div className="identifier">{el.proposal}</div>
+                    <div className="identifier">
+                      <p>{el.proposal}</p>
+                      <p
+                        onClick={() => {
+                          open();
+                          setModalState({
+                            proposal: el.proposal,
+                            correct: el.correct,
+                            totalSupply: el.totalSupply,
```

Comment:
this line crashes for me when clicking on "View Details"

Reply:
this fails on the most recent past requests, admin 86 and admin 87. probably need to set some kind of default values.

Reply:
These shouldn't even be displayed. Lemme filter them out -- they aren't technically Past Requests, and so don't have certain pieces of data.

I should probably do some kind of catch though, in case the data hasn't been properly indexed yet.